about.html

```diff
@@ -0,0 +1,82 @@
+                    <li><a href="#" class="header-icon">Logout</a></li>
+                </ul>  
+            </header>
+            <nav class="Nav-bar">
+                <a class="top-logo" id="logo" ><img src="assets/nav-bar-logo.jpg"/></a>
+                <ul class='nav-menu'>
+                    <li class="nav-item"><a class="nav-link" href="./index.html">Home</a> </li>
+                    <li class="nav-item"><a class="nav-link"  href="#">About</a></li>
+                    <li class="nav-item"><a class="nav-link"  href="">Booking</a></li>
+                    <li class="nav-item"><a class="nav-link"  href="">Hotels</a></li>
+                </ul>
+                <div class="hamburger-menu">
+                    <div class="bar"></div>
+                    <div class="bar"></div>
+                    <div class="bar"></div>
+                </div>
+            </nav>    
+        
+        <h1 class="heading">Hello Great Innovators!</h1>
+        <h5 class="heading-1">Welcome to the 2022 World Tech Innovators Forum</h5>
+        <p class="first-paragraph">The Innovators forum brings in the greatest tech brains from all over the world. come and share the experience!</p>
+
+        <div class="great"><span>Contact us for booking and inquiries</span><span> the World Tech Innovators forum 2022</span><span>TechInnovators@info.co.org</span></div>
+    
+        
+    </section>
+    <section>
+        <div class="logo-head">
+            <h4 class="logo-content">World Tech Innovators Forum</h4>
+            <h4 class="logo-content">Innovators Forum 2022 Logo</h4>
+            <div class="under-line"></div>
+        </div>
+        <div class="logo-detail">
+            <p class="logo-content-1">The Innovators forum logo was a gift from lp.inc.</p>
+            <p class="logo-content-1">designed for business purposes</p>
+        </div>
+    </section>
+    <div class="logo-photo"><img id="logo-picture" src="assets/logophoto.png" /></div>
+    <section class="logo-head">
+        <h4 class="logo-content">Past World Tech Innovator's forums</h4>
+        <div class="under-line"></div>
+        <p>Take a look at the past forums that took place in Washington DC and Los Angeles</p>
+        <div class="past-forums">
+            <div class="conference"><span id="conf-id">2020</span><span id="conf-id">Innovator's forum Washington DC</span></div>
+            <div class="conference-1"><span id="conf-id">2018</span><span id="conf-id">Innovator's forum Los Angeles</span></div>
+        </div>
+    </section>
+    <section class="partners">
+        <h4 class="logo-content">Partner</h4>
+        <div class="under-line"></div>
+        <div class="partner-details">
+            <div id="partner-name">Mozilla</div>
+            <div id="partner-name">Google</div>
+            <div id="partner-name">NAVER</div>
+            <div id="partner-name">Slack</div>
+            <div id="partner-name">IBM</div>
+        </div>
+    </section>
+    <footer>
```

Comment:
Please check and ensure that the footer of the `About page` has a dark background whilst that of the `Home page` is white. :+1:  Currently both have white backgrounds. :-1: